### PR TITLE
chore(main): update solidity doc links to latest

### PR DESCRIPTION
**Pull request by brachsterX**
Gets rid of current good-first-issue tasks
#16748 #16598 #16597

### CODE REVIEW

File: packages/protocol/docs/how_taiko_proves_blocks.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ struct BlockMetadata {
 
 ### Global Variables
 
-The following [**block level variables**](https://docs.soliditylang.org/en/v0.8.18/units-and-global-variables.html) are accessible to the EVM, but their values are not part of the MPT so we need a different way to verify their correctness.
+The following [**block level variables**](https://docs.soliditylang.org/en/v0.8.25/units-and-global-variables.html) are accessible to the EVM, but their values are not part of the MPT so we need a different way to verify their correctness.
```
Comment (dantaik) [inline packages/protocol/docs/how_taiko_proves_blocks.md:109]: change to use `https://docs.soliditylang.org/en/latest/units-and-global-variables.html` please.

File: packages/protocol/docs/tokenomics-whitepaper.tex [hunk 2]
```diff
@@ -52,7 +52,7 @@ \section*{References}
     \item \href{https://ethereum.org/en/developers/docs/layer-2-scaling/}{Ethereum Layer 1 and Layer 2 Integration}
     \item \href{https://ethereum.org/en/governance/}{Ethereum Governance Documentation}
     \item \href{https://ethereum.org/en/whitepaper/}{Ethereum Whitepaper}
-    \item \href{https://docs.soliditylang.org/en/v0.8.3/}{Solidity Documentation}
+    \item \href{https://docs.soliditylang.org/en/v0.8.25/}{Solidity Documentation}
```
Comment (dantaik) [inline packages/protocol/docs/tokenomics-whitepaper.tex:55]: also use latest here.

File: packages/relayer/processor/processor.go [hunk 3]
```diff
@@ -205,7 +205,7 @@ func InitFromConfig(ctx context.Context, p *Processor, cfg *Config) error {
 			return err
 		}
 
-		// only support one hop rn, add in array configs
+		// only support one hop right now, add in array configs
```
Comment (dantaik) [inline packages/relayer/processor/processor.go:208]: Could you undo this modification and change the title of the PR to something like "update solidity doc link to *latest*.

File: packages/protocol/docs/tokenomics-whitepaper.tex (hunk 2 above)
Comment (brachsterX) [reply packages/protocol/docs/tokenomics-whitepaper.tex:55]: 0.8.25 is already the latest

File: packages/relayer/processor/processor.go (hunk 3 above)
Comment (brachsterX) [reply packages/relayer/processor/processor.go]: done

**Review by dantaik — APPROVED** [review]

**Review by davidtaikocha — APPROVED** [review]

**Pull request merged**